### ISSUE TITLE
Please make CONCEPT_ID 32220 a Standard Concept

### ISSUE THREAD

**Issue by clairblacketer** [OHDSI/Vocabulary-v5.0#505]
There is no standard Visit concept for the idea "still in hospital" which is a concept we use to identify patients that are still in the hospital at the time of the data extract. There is a UB04 code for this, represented by the non-standard concept [32220](https://athena.ohdsi.org/search-terms/terms/32220). Can you please make this a standard concept?

**Comment by cgreich** [comment]
@clairblacketer: 

Is this trying to solve the problem of visit_end_date at the end of the database?

**Comment by Alexdavv** [comment]
@clairblacketer It looks like we've found a kind of solution [here](https://forums.ohdsi.org/t/themis-2-how-do-you-handle-missing-visit-end-dates/4308/31). Has something changed?

**Comment by clairblacketer** [comment]
So the solution suggested in the the thread has us put the OBSERVATION_PERIOD_END_DATE as the VISIT_END_DATE for patients still in the hospital. I agree with this assessment but there is still not a good way to determine which patients are still in the hospital and which patients were actually discharged on that day. As suggested in the forum post we would like to use this concept for that purpose. Regardless, this source code does not have a mapping to a standard concept so it makes sense to me to just make this one standard.

**Comment by Alexdavv** [comment]
> Option #2 should satisfy all use cases. It’s just against the rules right now. And I would like the rule to be changed to allow NULL Visit end_dates ONLY when a Visit is ongoing. If the Visit is not ongoing, then an end_date needs to be populated. Populating the date of ETL as the Visit end_date for an ongoing Visit is not necessary and will affect analysis.

@clairblacketer I thought it was agreed that the NULL end date would be an indicator for continuing visit.
But now I see it’s not.

@cgreich @chrisknoll @MelaniePhilofsky @ericaVoss Do you feel we agreed on something?

**Comment by chrisknoll** [comment]
For me, a null end date is fine provided the duration of visit can be found by using the observation period end date.  For cohort definitions, I can make queries substitute observation_period_end_date for when visit_end_date is null.

I was arguing for non-null end dates in visits due to that I had no way of determining the duration of a visit as of a certain 'data-freeze' in time.  Substituting observation_end_date for a null visit_end_date seemed like a reasonable compromise.

**Comment by clairblacketer** [comment]
Bringing this back up - @Alexdavv we still need this concept 32220 to be made standard

**Comment by chrisknoll** [comment]
Can we do a quick thought experiment using a visit_occurrence with concept_id 32220?

You have a cohort definition, entry events are visits....you want to characterize time in hospital.   Is the record with 32220 have a visit end date?   If we have existing phenotypes that look for IP visits, does a record not show up until the person is finished with their visit?  Those are just a couple of questions to get the juices flowing, but I would like to understand how this type of visit_occurrence record is used in practice.

**Comment by clairblacketer** [comment]
@chrisknoll this concept is more for hospitals that convert their data on a daily or weekly basis. The basis of this question is for how to identify patients that are still in the hospital at the time of conversion. The idea is that the visit_end_date is the date of conversion and the concept of 32220 would denote that they are still in the hospital. This would change to 9201 once the person is discharged and the visit_end_date becomes the date of discharge.

**Comment by clarkevans** [comment]
I do not think this is a vocabulary issue. It's a data model issue. I think a reasonable fix could be put into tools currently, and then this change made official in a later CDM.

**Comment by cgreich** [comment]
> as a NULL end date by itself doesn't solve the problem of "visit ended but the date is missing" vs "visit was ongoing at the time of ETL"

I never understood why this is a problem. We don't know the end date. Whether that is because we don't know **yet**, or whether we will never know, is irrelevant. We cannot use this visit in the calculation of visit duration or use it for setting the cohort end date.

I vaguely remember only one reason was for patient recruitment, where we want to know patients still in the hospital NOW. But there are other ways to figure that out. Just take all NULL end date visits and see when the start date was. If it was in 2020 the likelihood the patient is still in the hospital is pretty small.

**Comment by kkondratieff** [comment]
@cgreich In the VA, we do actually have records from long-term care patients who are still in an in-patient setting that they entered in 2020 (or even before). They aren't very common, and it is possible some of them are actually cases where we simply never received a discharge date for them, but given the nature of care that the VA pays for and supervises, it's possible to have such long-lasting ongoing visits.

**Comment by chrisknoll** [comment]
Hi.  I agree with @kkondratieff that the idea of 'guessing' if a visit is too long they must have been discharged is a bad idea.

It's been a while since I reviewed my comments on this, but I'm in mostly agreement with my prior statements.  But how it's evovled a bit is how it seems to be trying to overload the meaning of a column:  visit end date being null means 'ongoing visit'.   The visit concept ID being switched from In Pateint or ER will switch from 'On Going' to 'ER' afterwards.  I think this is not proper.

If we need to know if a visit is ongoing as of ETL time, I'd add a column to the model to track that status (visit_is_active) or something like that.

The reason for my change of heart on what I've said earlier was that I thought that we'd always be able to interpret visit_end_date as ongoing, but due to Christian's comment that sometimes we just don't get it (and since it's required in the model, I guess the ETL'ers need to guess, but from the tools perspective it's always there).  If we're going to allow nulls to exist now, and ETL'ers no longer need to make the determination about when they think that visits end, and therefore some visits look like that are ongoing because the ETLers don't provide an end date for missing visit ends, i think all that is too much to account for and puts all the burdeon on tools to think for the ETL'er.  Therefore, if we want to distinguish between ongoing and completed visits, add a column to support that.
